tests/conftest.py

```diff
@@ -111,6 +111,7 @@ def fast_rolling_stock() -> int:
 @pytest.fixture
 def west_to_south_east_path(small_infra: Infra, fast_rolling_stock: int) -> TrainPath:
     """west_to_south_east_path screenshot in `tests/README.md`"""
+    response = requests.post(f"{EDITOAST_URL}infra/{small_infra.id}/load")
     response = requests.post(
```

Comment:
The first `response` variable is overwritten. You probably shouldn't store the result in a variable, and instead `.raise_for_status()` to raise an error if the request failed